src/components/MenuItem.tsx

```diff
@@ -429,6 +433,14 @@ function MenuItem(
         return parser.replace(title, {shouldEscapeText});
     }, [title, shouldParseTitle, shouldEscapeText]);
 
+    const helperHtml = useMemo(() => {
+        if (!helperText || !shouldParseTitle) {
```

Comment:
A little confused, why do we need to check `shouldParseTitle`. Shouldn't it be `shouldParseHelperText` ?